disperser/common/blobstore/blob_metadata_store.go

```diff
@@ -377,3 +400,46 @@ func UnmarshalBlobMetadata(item commondynamodb.Item) (*disperser.BlobMetadata, e
 
 	return &metadata, nil
 }
+
+func converTypeAttributeValuetToExclusiveBlobStoreStartKey(exclusiveStartKeyMap map[string]types.AttributeValue) (*disperser.ExclusiveBlobStoreStartKey, error) {
```

Comment:
Do we need a separate struct type for `ExclusiveBlobStoreStartKey`? 
Would it be ok to pass around exclusiveStartKey as `Key` type?

Reply:
understood.
nit: can we make the method name bit concise? reminds me of Java 😭